src/Components/SavedSearchAlert/Components/ConfirmationStepFooter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, Flex, SkeletonBox } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { FC } from "react"
+import { graphql } from "react-relay"
+import {
+  ConfirmationStepFooterQuery,
+  ConfirmationStepFooterQuery$data,
+} from "__generated__/ConfirmationStepFooterQuery.graphql"
+import { RouterLink } from "System/Router/RouterLink"
+import { useTranslation } from "react-i18next"
+
+interface ConfirmationStepFooterProps {
+  me: ConfirmationStepFooterQuery$data["me"]
+}
+
+const ConfirmationStepFooter: FC<ConfirmationStepFooterProps> = ({ me }) => {
+  const { t } = useTranslation()
+  const savedSearch = me?.savedSearch
+
+  return (
+    <Flex flexDirection={["column", "row"]}>
+      <Button
+        width="100%"
+        mr={[0, 1]}
+        mb={[1, 0]}
```

Comment:
Just use `gap` on flex instead of these margins

Reply:
TIL gap 👍

Reply:
It's *so* useful!